common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/Worker.cs

```diff
@@ -79,14 +79,14 @@ public class Worker : IWorker, IDisposable {
             await _lock.WaitAsync();
             try {
                 if (_cts != null) {
-                    _logger.Warning("Worker already running");
+                    _logger.Verbose("Worker already running");
                     return;
                 }
 
                 _cts = new CancellationTokenSource();
                 _heartbeatTimer.Change(TimeSpan.Zero, Timeout.InfiniteTimeSpan);
 
-                _logger.Information("Worker WorkerId: {WorkerId} {@Capabilities}",
+                _logger.Verbose("Worker {WorkerId}: {@Capabilities}",
```

Comment:
>Verbose [](start = 24, length = 7)

that's something that happens typically once during runtime. if happens more often might be a god indication for debugging

Reply:
36 hits, will revert